docker/web/Dockerfile

```diff
@@ -41,11 +39,12 @@ RUN wget https://getcomposer.org/download/1.7.3/composer.phar -O /usr/local/bin/
 
 # js stuff
 RUN curl -sL https://deb.nodesource.com/setup_11.x | bash -
-RUN apt-get update && apt-get install -y nodejs
-
 RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+
 RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
```

Comment:
I think current _stable_ is _buster_.

Reply:
Only docs for installation are about stable.
https://yarnpkg.com/en/docs/install#debian-stable